packages/app/src/plugins.ts

```diff
@@ -44,3 +44,4 @@ export { plugin as Buildkite } from '@roadiehq/backstage-plugin-buildkite';
 export { plugin as Search } from '@backstage/plugin-search';
 export { plugin as Org } from '@backstage/plugin-org';
 export { plugin as Kafka } from '@backstage/plugin-kafka';
+export { plugin as SplunkOnCall } from '@backstage/plugin-splunk-on-call';
```

Comment:
Again, would suggest we may not want to change this globally for the example app, but if needed adding it to the README for the plugin setup instructions?